elevation_mapping/src/sensor_processors/SensorProcessorBase.cpp

```diff
@@ -158,5 +163,37 @@ void SensorProcessorBase::removePointsOutsideLimits(
   ROS_DEBUG("removePointsOutsideLimits() reduced point cloud to %i points.", (int) pointClouds[0]->size());
 }
 
+bool SensorProcessorBase::cleanPointCloud(const pcl::PointCloud<pcl::PointXYZRGB>::Ptr pointCloud)
+{
+	pcl::PassThrough<pcl::PointXYZRGB> passThroughFilter;
+	pcl::PointCloud<pcl::PointXYZRGB> tempPointCloud;
```

Comment:
Please check the formatting again, here and in other places tabs/spaces are mixed up.